stable/spark/values.yaml

```diff
@@ -8,31 +8,37 @@ Spark:
 
 Master:
   Name: master
-  Image: "k8s.gcr.io/spark"
-  ImageTag: "1.5.1_v3"
+  Image: "dbanda/spark2.4"
```

Comment:
Why are we changing the container image source?

Reply:
The "k8s.gcr.io/spark" was with an outdated version of spark . We built a docker image with spark 2.4.0 . I will pushed a PR to move the docker image to MCR as more permanent location for the image. It should go through in the next few days.

Reply:
@dbanda @mhamilton723 I see this has already been done 👍 https://github.com/Azure/mmlspark/tree/master/tools/helm/docker_images Will those images be listed here? https://hub.docker.com/r/microsoft/mmlspark

Reply:
Right now the images are not on hub.docker.com yet. They are on mcr.microsoft.com/mmlspark (microsoft's  repo for public images). My latest push has the correct image tags.